apps/stages/src/transformation/application-service/transformer-flux-jobteaser.usecase.ts

```diff
@@ -10,14 +13,17 @@ export class TransformerFluxJobteaser implements Usecase {
 	constructor(
 		private readonly offreDeStageRepository: OffreDeStageRepository,
 		private readonly convertirOffreDeStage: Convertir,
+		private readonly dateService: DateService,
 	) {
 	}
 
 	public async executer(flux: FluxTransformation): Promise<void> {
 		const contenuDuFlux = await this.offreDeStageRepository.recuperer<Jobteaser.Contenu>(flux);
 
-		const contenuTransforme: Array<UnJeune1Solution.OffreDeStage>
-			= contenuDuFlux.jobs.job.map((job: Jobteaser.OffreDeStage) => this.convertirOffreDeStage.depuisJobteaser(job));
+		const contenuFiltre: Array<Jobteaser.OffreDeStage> = contenuDuFlux.jobs.job.filter((job: Jobteaser.OffreDeStage) => {
+				return !job.start_date || DateTime.fromISO(job.start_date) >= DateTime.fromJSDate(this.dateService.maintenant()).minus({ "month": 1 });
+		});
```

Comment:
tu peux extraire this.dateService.maintenant()).minus({ "month": 1 }) dans une constante pour mieux lire le code